pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -47,6 +54,7 @@
 public class SegmentStatusChecker {
   private static final Logger LOGGER = LoggerFactory.getLogger(SegmentStatusChecker.class);
   private static final int SegmentCheckerDefaultIntervalSeconds = 120;
+  private static final long waitForPushTime = 3600 * 1000;
```

Comment:
Can you please rename this var to waitForPushTimeMs (if it is milliseconds, which I think it is)? thanks